src/modules/commander/PreflightCheck.cpp

```diff
@@ -391,8 +391,8 @@ bool preflightCheck(orb_advert_t *mavlink_log_pub, bool checkMag, bool checkAcc,
 	// all the sensors are supported
 	PX4_WARN("Preflight checks always pass on Snapdragon.");
 	return true;
-#elif defined(__LINUX)
-	PX4_WARN("Preflight checks always pass on Linux (RPI).");
+#elif defined(__PX4_POSIX_RPI)
```

Comment:
No, if you see in the history this check was only recently introduced, together with other RPI changes. It was never intended to be for linux in general.